ui/pages/BlockCountdown.tsx

```diff
@@ -38,10 +38,22 @@ const BlockCountdown = ({ hideCapybaraRunner }: Props) => {
     },
   });
 
+  // Mock remaining blocks time
+  if(data) {
+    data.result = {
+      EstimateTimeInSec: `${2 * Number(data?.result?.RemainingBlock)}`,
+      CountdownBlock: data?.result?.CountdownBlock as string,
+      CurrentBlock: data?.result?.CurrentBlock as string,
+      RemainingBlock: data?.result?.RemainingBlock as string
+    };
+    console.log("bloo", data?.result?.RemainingBlock);
+  }
+
```

Comment:
Maybe we can expose an env variable controlling the mocking?

Reply:
Done

Reply:
I don't think the frontend should do the mocking. If at all, it should be done on the back-end, but I think once we have testnets with reliable block production we won't need that at all.

I'd like to keep this change minimal for easier removal in the future.

Reply:
I don't think the frontend should do the mocking. If at all, it should be done on the back-end, but I assume once we have testnets with reliable block production we won't need that at all.

I'd like to keep this change minimal for easier removal in the future.

Reply:
Done